namespaces/live.cloud-platform.service.justice.gov.uk/grc-prod/resources/route53paas.tf

```diff
@@ -73,6 +73,6 @@ resource "aws_route53_record" "dashboard_apply_gender_recognition_certificate" {
   type    = "CNAME"
   ttl     = 300
   records = [
```

Comment:
@mark-butler-solirius , if you want to point the url `"dashboard.apply-gender-recognition-certificate.service.gov.uk"` to the app in Cloud Platform, please remove this whole section.

The ingress in your namespace will automatically generate the A record related to that url